docs-src/python/backgroundcheck-boilerplate-workflow-details.md

```diff
@@ -0,0 +1,30 @@
+---
+id: backgroundcheck-boilerplate-workflow-details
+title: Boilerplate Workflow code
+sidebar_label: Workflow code
+description: In the Temporal Python SDK programming model, a Workflow Definition is defined as a class.
+tags:
+- go sdk
+- workflow
+- developer guide
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-python/blob/bgc/backgroundcheck_boilerplate/workflows/backgroundcheck_dacx.py. -->
+
+Use the `@workflow.defn` decorator on the `BackgroundCheck` class to identify a Workflow.
+
+Use the `@workflow.run` to mark the entry point method to be invoked. This must be set on one asynchronous method defined on the same class as `@workflow.defn`.
+
+Run methods have positional parameters.
+
+By default Workflows are run in a sandbox to help avoid non-deterministic code.
+If a call that is known to be non-deterministic is performed, an exception will be thrown in the workflow which will "fail the task" which means the workflow will not progress until fixed.
+Import your Activity code into the Workflow Definition using the `with [workflow.unsafe.imports_passed_through()](https://python.temporal.io/temporalio.workflow.unsafe.html#imports_passed_through)` context manager.
+
+To spawn an [Activity Execution](https://python.temporal.io/temporalio.workflow.html#execute_activity), call `execute_activity()` inside your Workflow Definition.
+This API is available from the `workflow` module from the `temporalio` package.
+The `execute_activity` API call requires either `schedule_to_close_timeout` or `start_to_close_timeout`.
+
+In this example, pass in the Activity name, `ssn_trace_activity` and an argument, `ssn`.
```

Comment:
I do not see an example in this file (trying to understand the value of breaking up the files)

Reply:
Surfaced this in a larger convo.